cudamapper/samples/sample_cudamapper.cpp

```diff
@@ -138,11 +149,12 @@ void process_batch(std::vector<IndexDescriptor>& query_index_descriptors,
 int main(int argc, char** argv)
 {
     // parse command line options
-    int c      = 0;
-    bool help  = false;
-    bool print = false;
+    int c               = 0;
+    bool help           = false;
+    bool print          = false;
+    OutputFormat format = OutputFormat::PAF;
 
-    while ((c = getopt(argc, argv, "hp")) != -1)
+    while ((c = getopt(argc, argv, "hpSB")) != -1)
```

Comment:
How is `p` used?

Reply:
p is as a switch to print to stdout (or print nothing at all). It's a flag that's used in all GW samples